src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -173,19 +156,60 @@ CUP mode:
         </property>
        </widget>
       </item>
-      <item row="2" column="0">
-       <widget class="QLabel" name="labelTimeDisplay">
+      <item row="4" column="1" colspan="2">
+       <widget class="QCheckBox" name="checkBoxDisallowLoadToPlayingDeck">
         <property name="text">
-         <string>Time Format</string>
+         <string>Do not load tracks into playing decks</string>
         </property>
        </widget>
       </item>
-      <item row="2" column="1">
-       <widget class="QComboBox" name="comboBoxTimeFormat"/>
+      <item row="5" column="0">
+       <widget class="QLabel" name="labelCloneDeckOnLoadDoubleTap">
+        <property name="text">
+         <string>Clone Deck</string>
```

Comment:
```suggestion
         <string>Clone deck</string>
```
for consistency with the other labels. Could you change "Time Format" to "Time format" too?

Reply:
I can do so, then at least the Decks page is cconsistent.
But there are lots of inconsistencies in the other pages. This should be fixed another time